### ISSUE TITLE
MD005: False-positive for unordered list nested in ordered list

### ISSUE THREAD

**Issue by patrickhousley** [DavidAnson/markdownlint#247]
The below markdown, when linted, violates rule MD005 for the nested unordered list part. I have tried 3 and 4 spaces. The only way to make it work is to de-indent the unordered list but then it is no longer nested inside the ordered list.

```markdown
1. Select the Group or project to create the board.
1. Select "Issues" from Left Column menu
1. Select "Board" from Issue Menu list
1. Open Dropdown for Switch board at top of page.
    *The default is Developement*
1. Select "Create New Board"
1. Enter Board name
1. Expand Board scope.
1. Enter desired scope.
    - Milestone
    - Labels
    - Assignees
    - Weights
1. Click "Create Board"
```

```bash
09:59 $ mdl content
content/docs/framework/Agile_Documentation.md:87: MD005 Inconsistent indentation for list items at the same level
content/docs/framework/Agile_Documentation.md:88: MD005 Inconsistent indentation for list items at the same level
content/docs/framework/Agile_Documentation.md:89: MD005 Inconsistent indentation for list items at the same level
content/docs/framework/Agile_Documentation.md:90: MD005 Inconsistent indentation for list items at the same level
```

**Comment by DavidAnson** [comment]
`mdl` is the name used by the Ruby implementation, could you check if you’re using that? https://github.com/markdownlint/markdownlint

Your sample produces no errors with the Node implementation (this project): https://dlaa.me/markdownlint/#%25m%23%20Issue%20247%0A%0A1.%20Select%20the%20Group%20or%20project%20to%20create%20the%20board.%0A1.%20Select%20%22Issues%22%20from%20Left%20Column%20menu%0A1.%20Select%20%22Board%22%20from%20Issue%20Menu%20list%0A1.%20Open%20Dropdown%20for%20Switch%20board%20at%20top%20of%20page.%0A%20%20%20%20*The%20default%20is%20Developement*%0A1.%20Select%20%22Create%20New%20Board%22%0A1.%20Enter%20Board%20name%0A1.%20Expand%20Board%20scope.%0A1.%20Enter%20desired%20scope.%0A%20%20%20%20-%20Milestone%0A%20%20%20%20-%20Labels%0A%20%20%20%20-%20Assignees%0A%20%20%20%20-%20Weights%0A1.%20Click%20%22Create%20Board%22%0A

**Comment by patrickhousley** [comment]
Sorry @DavidAnson I threw this in the wrong project.

**Comment by patrickhousley** [comment]
For reference, I moved this to an already open issue: https://github.com/markdownlint/markdownlint/issues/296
